fedimint-client/src/db.rs

```diff
@@ -52,6 +54,12 @@ pub enum DbKeyPrefix {
     EventLog = fedimint_eventlog::DB_KEY_PREFIX_EVENT_LOG,
     UnorderedEventLog = fedimint_eventlog::DB_KEY_PREFIX_UNORDERED_EVENT_LOG,
 
+    DatabaseVersion = fedimint_core::db::DbKeyPrefix::DatabaseVersion as u8,
```

Comment:
This thing immediately cough the fact that our db key prefix enums did not contain *all* prefixes, and the fact that we could by accident have conflicts.